app/src/test/java/com/willowtree/vocable/utils/FaceTrackingPermissionsTest.kt

```diff
@@ -1,19 +1,36 @@
 package com.willowtree.vocable.utils
 
+import android.content.pm.PackageManager
+import androidx.core.content.ContextCompat
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkStatic
 import kotlinx.coroutines.ExperimentalCoroutinesApi
 import kotlinx.coroutines.flow.first
 import kotlinx.coroutines.test.advanceUntilIdle
 import kotlinx.coroutines.test.runTest
 import org.junit.Assert.assertEquals
 import org.junit.Assert.assertFalse
 import org.junit.Assert.assertTrue
+import org.junit.Before
 import org.junit.Test
 
 @OptIn(ExperimentalCoroutinesApi::class)
 class FaceTrackingPermissionsTest {
 
+    @Before
+    fun setup() {
+        mockkStatic(ContextCompat::class)
+        every {
+            ContextCompat.checkSelfPermission(any(), any())
+        } returns PackageManager.PERMISSION_DENIED
+    }
+
     private fun createFaceTrackingPermissions(sharedPreferences: IVocableSharedPreferences): IFaceTrackingPermissions {
-        return FaceTrackingPermissions(sharedPreferences)
+        return FaceTrackingPermissions(
+            sharedPreferences = sharedPreferences,
+            activity = mockk(relaxed = true)
+        )
```

Comment:
holler if there's a better approach since I know relaxed mocks are probably a code smell, but an activity is required in this class now and doing interface shenanigans didn't feel worth the time/complexity to me